internal/client/github.go

```diff
@@ -83,8 +84,12 @@ func (c *githubClient) CreateFile(
 
 func (c *githubClient) CreateRelease(ctx *context.Context, body string) (releaseID int, err error) {
 	var release *github.RepositoryRelease
+	releaseTitle, err := name.ForTitle(ctx)
```

Comment:
I'll have to refactor this template thing in the name package... it is starting to look bad I guess